profiles/events.py

```diff
@@ -16,16 +16,23 @@ def send_update_events(publisher: EventPublisher) -> Callable[..., None]:
     def event_handler(sender: Flask, changes: tuple) -> None:  # pylint:disable=unused-argument
         ids = []
 
+        LOGGER.debug('Processing event(s)')
```

Comment:
Speaking of the log level, this could easily be `info` as it should be left turned on in `profiles--prod` for a long time. We have no good log at the SNS level of what is going on, only some monitoring on the SQS queues that listen to the topic.

Reply:
My rule of thumb is `if it doesn't fill the disk, then it's info`